source/boards/get-started-with-boards.rst

```diff
@@ -0,0 +1,79 @@
+Get Started with Mattermost Boards
+==================================
+
+Access Mattermost Boards by selecting **Boards** from the **Product list** |product-list| icon.
+
+.. |product-list| image:: ../images/products_E82F.svg
+  :height: 24px
+  :width: 24px
+  :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.
+
+Boards are used for planning projects collaboratively. You can access the list of channels and their boards from the drop-down menu in the left-hand sidebar. Boards are made up of cards, and cards can have a wide range of purposes.
+
+See our `Mattermost Boards <https://docs.mattermost.com/guides/boards.html>`__ documentation for details on the many ways you can use Channels to enable team collaboration.
+
+Create a board
+--------------
+
+When you first start using Boards, you need to create a board. You can select a template from the options provided or you can create an empty board. The only difference is that a template will be populated with sample cards and properties and in an empty template you start from scratch.
+
+To find a board, open Boards and select the drop-down menu in the sidebar. Each board is linked to the channel it was created in. This includes direct messages and group messages.
```

Comment:
```suggestion
To find a board, open Boards and select the drop-down menu in the sidebar. You can search for a workspace or browse through all existing workspaces. Each workspace is linked to the channel it was created in. This includes direct messages and group messages.
```

Reply:
Note: We will need to change the terminology to "boards" instead of "workspaces" after we ship the new permissions system in v6.6.

Reply:
Thanks for the heads up! :)